chandra_aca/drift.py

```diff
@@ -173,3 +174,42 @@ def get_aca_offsets(detector, chip_id, chipx, chipy, time, t_ccd):
     ddz = dz_chip - dz_pred
 
     return ddy, ddz
+
+
+def get_web_zero_offset_table():
+    """
+    Get official SOT MP zero offset aimpoint table:
+
+    :returns: zero offset aimpoint table as astropy.Table
+    """
+    ZOA = "https://icxc.harvard.edu/mp/html/aimpoint_table/zero_offset_aimpoints.txt"
+    page = requests.get(ZOA)
+    table_lines = page.content.strip().split("\n")
+    return Table.read(table_lines, format="ascii")
+
+
+def get_target_aimpoint(date, cycle, detector, too=False, zero_offset_table=None):
+    """
+    Given date, proposal cycle, and detector, return aimpoint chipx, chipy, chip_id
+
+    :param date: observation date
+    :param cycle: proposal cycle of observation
+    :param detector: target detector
+    :param too: boolean. If target is TOO use current cycle not proposal cycle.
+
+    :returns: astropy table row of chipx, chipy, chip_id
+    """
+    if zero_offset_table is None:
+        zero_offset_table = get_web_zero_offset_table()
+    # Get entries for this detector before the 'date' given
+    filtered_table = zero_offset_table[
```

Comment:
What is `DateTime(date).date[:10]` supposed to be giving me?   My guess is that you are trying to simplify the date comparisons, but I've got:

```
In [14]: DateTime().date[:10]
Out[14]: '2016:235:1'
```